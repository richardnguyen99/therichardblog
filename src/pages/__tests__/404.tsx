/**
 * @jest-environment jsdom
 */

import * as React from "react";
import { render } from "@testing-library/react";

import ErrorPage from "../404";

describe("<ErrorPage />", () => {
  it("should match snapshot", () => {
    const { container } = render(<ErrorPage />);

    expect(container).toMatchSnapshot();
  });
});
