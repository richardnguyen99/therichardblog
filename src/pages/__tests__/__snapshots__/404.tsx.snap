// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage /> should match snapshot 1`] = `
<div>
  <main
    style="color: rgb(35, 33, 41); padding: 96px; font-family: -apple-system, Roboto, sans-serif, serif;"
  >
    <title>
      Not found
    </title>
    <h1
      style="margin-top: 0px; margin-bottom: 64px; max-width: 320px;"
    >
      Page not found
    </h1>
    <p
      style="margin-bottom: 48px;"
    >
      Sorry
       
      <span
        aria-label="Pensive emoji"
        role="img"
      >
        ðŸ˜”
      </span>
       
      we couldnâ€™t find what you were looking for.
      <br />
      <br />
      <a
        href="/"
      >
        Go home
      </a>
      .
    </p>
  </main>
</div>
`;
